src/python/turicreate/test/test_image_classifier.py

```diff
@@ -84,19 +88,24 @@ class ImageClassifierTest(unittest.TestCase):
     def setUpClass(
         self,
         model="resnet-50",
+        feature="resnet-50_deep_features", 
```

Comment:
I don't think this right. If the class name doesn't end in `WithDeepFeature`, then I think it should be `feature="awesome_image"`. This code is only calling `get_deep_features` when `self.feature != "awesome_image"`.

I think you need to make this change to all of your test classes.

Reply:
Changed the deep feature column name in test data to the one suggested by you and made all the requested changes.

Reply:
I think you misunderstood my previous comment. I was **not** suggesting you need to change the name of the feature. I was basically just trying to say that it's important for the name of the class to represent what that class is actually testing.

For example, if the name of the class is `ImageClassifierResnetTestWithDeepFeatures`, then it should be `feature="resnet-50_WithDeepFeature"` or `feature="resnet-50_deep_features",` but not `feature="awesome_image",`. Similarly, if the name of the class is `VisionFeaturePrintSceneTest` then it should not be creating the model from deep features, so it should be `feature="awesome_image",` not `feature="VisionFeaturePrint_Scene_WithDeepFeature"`.

Does that make sense? Let me know if you have any questions.

Reply:
Oh yeah, you are right I did not see it there. It was correct before but during one of my recent pushes I changed it to test something and forgot to revert it back. My bad, will change it and push it. 
And about changing the name of feature columns in the test data in `test_image_classifier.py` and `test_image_similarity.py` to have a suffix `_WithDeepFeature` instead of `_deep_features` makes more sense to me. So I am keeping that change as it is.